### DISCUSSION TITLE
Bug in Settings/Aliases

### DISCUSSION THREAD

**Discussion by whisperdancer** [simple-login/app#1384] [Self-hosting question]
Self-hosting v4.6.2.beta. Found a bug. In settings/Aliases

I changed selection to random words from our dictionary and update, works fine.
Then, change back to Random combination of 5 letter and digits and update

Issue: It doesn't change. Creating random aliases are still created using random words from your dictionary rather than the Random combination of 5 letters and digits

**Comment by nguyenkims** [comment]
@whisperdancer do you see any error in the container log? I don't remember any change that could affect this part.

**Reply by whisperdancer** [reply]
@nguyenkims I change the setting from one to the other. Each random alias I create always is the random words from your dictionary no matter which of the 2 settings is selected. No errors in the sl-app log that I can see.

Setting: Random combination of 5 letters and digits
Random alias created: newfangled_klatch@<mydomain.com>

**Reply by nguyenkims** [reply]
I did these steps and the random alias is correctly created:

1. Change random alias to UUID
 
<img width="454" alt="image" src="https://user-images.githubusercontent.com/3167780/198969874-d53da379-a7fa-470a-8c1d-25a83d4a9827.png">

2. click on the random alias button, the new alias is indeed UUID
<img width="196" alt="image" src="https://user-images.githubusercontent.com/3167780/198969958-8e1df300-6f15-463e-b152-3f23d8c0654e.png">

**Reply by whisperdancer** [reply]
Unsure how you uploaded an image. I can't figure out how to do that so here are my settings:

Settings/Aliases
Based on random word
Random combination of 5 letters and 5 digits

On Aliases screen, create new random alias based on random words and the address created is this:
perquisite_ngunnawal@<mydomain.com>

This is not using a random 5 letters and digits to create the random alias. It is using the other option "Random words from our dictionary" to create the alias.
